### PR TITLE
data: himanshubalani

### PR DESCRIPTION
**Pull request by himanshubalani**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3634"></a>

### CODE REVIEW

File: data/himanshubalani.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/himanshubalani/",
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Befriend me on Discord",
+      "url": "https://discord.com/users/759807486831099928",
```
Comment (Panquesito7) [inline data/himanshubalani.json:20]: Not sure that link will work. I've tried similar stuff in the past, and it didn't work. 🤔

**Review by Panquesito7 — APPROVED** [review]
Overall, it's looking good. Thanks! 🎉

File: data/himanshubalani.json (hunk 1 above)
Comment (himanshubalani) [reply data/himanshubalani.json:20]: Oh it does work if you're logged in Discord.

**Review by Panquesito7 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**